packages/simcore-sdk/src/simcore_sdk/node_ports_v2/port.py

```diff
@@ -146,13 +146,17 @@ def __init__(self, **data: Any):
             self._py_value_converter = Path
 
         elif self.property_type == "ref_contentSchema":
-            self._py_value_type = (int, float, bool, str, list, dict)
-            self._py_value_converter = lambda v: v
+            self._py_value_type = (int, float, bool, str, list[Any], dict[str, Any])
+
+            def _converter(value: ItemConcreteValue) -> ItemConcreteValue:
+                return value
+
+            self._py_value_converter = _converter
 
         else:
             assert self.property_type in TYPE_TO_PYTYPE  # nosec
 
-            self._py_value_type = TYPE_TO_PYTYPE[self.property_type]
+            self._py_value_type = (TYPE_TO_PYTYPE[self.property_type],)
```

Comment:
are you sure about this one?

Reply:
well at least I can tell you that the way this was done before was wrong and not according to the type hints that are set on that variable

Reply:
actually wondering if that thing is even used... I will remove it